tests/nexus/README.md

```diff
@@ -0,0 +1,29 @@
+# `Nexus` test framework
+
+`Nexus` is a test framework for OpenThread testing.
```

Comment:
Is Nexus something you created yourself?  Should we just use Nexus (without code font) to refer to the application and `nexus` (lowercase) to refer to the binary/command itself?

Reply:
Yes. Good idea. Changed in the docs to remove the back-ticks.